hls4ml/utils/config.py

```diff
@@ -413,7 +413,7 @@ def make_layer_config(layer):
 
 
 def config_from_onnx_model(
-    model, granularity='name', backend=None, default_precision='ap_fixed<16,6>', default_reuse_factor=1
+    model, granularity='name', backend=None, default_precision='fixed<16,6>', default_reuse_factor=1, max_precision=None
```

Comment:
Out of curiosity, why this change of removing the `ap_`? Should I do the same in https://github.com/fastmachinelearning/hls4ml/pull/1112?